packages/react-i18n/src/i18n.ts

```diff
@@ -500,6 +510,27 @@ export class I18n {
     });
   }
 
+  private humanizeFutureDate(date: Date, options?: Intl.DateTimeFormatOptions) {
+    const time = this.formatDate(date, {
+      ...options,
```

Comment:
Should this be moved two lines down so consumers can override hour and minute settings?

Reply:
The time implementation in `Humanize` is the same as before the PR.

I think it is a good idea. If allowing hour and minute overrides is something we want to do in `Humanize` - could it be done in a follow up PR?

Reply:
I'm down with that 👍

Reply:
I think we probably want to enforce good practice here. I think I prefer the current implementation for this reason but maybe @patsissons have other thoughts.